spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/repository/it/SpannerRepositoryIntegrationTests.java

```diff
@@ -167,6 +167,10 @@ public void queryMethodsTest() {
 		List<Trade> buyTradesRetrieved = this.tradeRepository
 				.annotatedTradesByAction("BUY");
 		assertThat(buyTradesRetrieved).containsExactlyInAnyOrderElementsOf(trader1BuyTrades);
+		assertThat(buyTradesRetrieved.get(0).getId()
+				.compareTo(buyTradesRetrieved.get(1).getId())).isPositive();
```

Comment:
why isPositive relevant?

Reply:
This is to check that the results came back in sorted order. IsPositive is checking that it came back in DESC order

Reply:
Oh I see. Maybe just use assertThat(..).isGreater{Less}Than(..)?